etna/search/tests/test_forms.py

```diff
@@ -55,14 +73,20 @@ def test_various_opening_date_inputs_is_valid(self):
                 form = CatalogueSearchForm(form_data)
                 self.assertTrue(form.is_valid(), label)
 
-    def test_opening_start_date_before_opening_end_date_is_invalid(self):
+    def test_start_date_after_end_date_is_invalid(self):
```

Comment:
Similar thing here. A separate test for each range would be clearer

Reply:
Same answer on separate comment

Reply:
Done. [here](https://github.com/nationalarchives/ds-wagtail/pull/991/commits/f17e917ada03a73c8a6062a5abaadf9156cf579d).